styles/main.scss

```diff
@@ -1405,4 +1405,49 @@ blockquote p:last-child {
 .remark-highlight {
   overflow: auto;
   max-width: calc(100vw - 30px);
+  position: relative;
+}
+
+.btn_copy {
```

Comment:
@ntorbinskiy do we name classes like this in the project?

Reply:
@1alexvash @alexander-vashchuk 
You mean, that I use <<_>>, not <<->> in the names of classes?